### PR TITLE
Update favicon & title

### PR DESCRIPTION
**Pull request by VinneyJ**
1. Change Media Cloud favicon to Civic Signal Favicon
2. Change site title

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,3 @@
+BUILDPACK_URL=
+
+SENTRY_DSN=
```
Comment (kilemensi) [inline .env.sample:3]: Is this file used anywhere?

File: server/templates/index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
   <meta name="viewport"
     content="width=device-width, initial-scale=1, user-scalable=0, maximum-scale=1, minimum-scale=1"
   >
-  <link rel="icon" type="image/png" href="{{ url_for('static', filename='favicon.png') }}" />
+  <link rel="icon" type="image/png" href="{{ url_for('static', filename='favicon-32x32.png') }}" />
```
Comment (kilemensi) [inline server/templates/index.html:10]: Include support for all favicon types:

```suggestion
  <link rel="apple-touch-icon" sizes="180x180" href={{ url_for('static', filename='apple-touch-icon.png') }}">
  <link rel="icon" type="image/png" sizes="32x32" href="{{ url_for('static', filename='favicon-32x32.png') }}">
  <link rel="icon" type="image/png" sizes="16x16" href="{{ url_for('static', filename='favicon-16x16.png') }}">
  <link rel="manifest" href="{{ url_for('static', filename='site.webmanifest') }}">
  <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}">
  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#022069">
  <meta name="msapplication-TileColor" content="#2b5797">
  <meta name="theme-color" content="#022069">
```

[favicon_package_v0.16.zip](https://github.com/CodeForAfrica/web-tools/files/15496918/favicon_package_v0.16.zip)

File: src/components/source/mediaSource/SourceFeedContainer.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ import TabSelector from '../../common/TabSelector';
 
 const localMessages = {
   pageTitle: { id: 'source.feeds.pageTitle', defaultMessage: 'Feeds' },
-  title: { id: 'source.feeds.title', defaultMessage: '{name} | Source Feeds | Media Cloud' },
+  title: { id: 'source.feeds.title', defaultMessage: '{name} | Source Feeds | Civic Signal ' },
```
Comment (kilemensi) [inline src/components/source/mediaSource/SourceFeedContainer.js:24]:
```suggestion
  title: { id: 'source.feeds.title', defaultMessage: '{name} | Source Feeds | Civic Signal' },
```

File: src/resources/messages.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 
 const messages = {
 
-  suiteName: { id: 'suite.name', defaultMessage: 'Media Cloud' },
+  suiteName: { id: 'suite.name', defaultMessage: 'Civic Signal ' },
```
Comment (kilemensi) [inline src/resources/messages.js:4]:
```suggestion
  suiteName: { id: 'suite.name', defaultMessage: 'Civic Signal' },
```

**Review by kilemensi — COMMENTED** [review]
👍🏽 

----

1. Build is failing
2. Update the title on [`index.html`](https://github.com/CodeForAfrica/web-tools/blob/38d953b466cdf04b5fc06c537fd73d6997f7f787/server/templates/index.html#L14) as well.

**Comment by kilemensi** [comment]
Looking at the site, the correct spelling is "CivicSignal" and not "Civic Signal" @VinneyJ

**Comment by VinneyJ** [comment]
Oh, thanks @kilemensi changing it ASAP.

**Comment by kilemensi** [comment]
PS: Where is the Google Analytics tag @VinneyJ ? I expect it to be in the [`index.html`](https://github.com/CodeForAfrica/web-tools/blob/38d953b466cdf04b5fc06c537fd73d6997f7f787/server/templates/index.html#L14) after `title`.

**Pull request closed without merging**